### ISSUE TITLE
[bug?] android scroll issue

### ISSUE THREAD

**Issue by derornos** [zzorba/ArkhamCards#258]
Drag action to scroll up should bring the page down, but it works the other way around. (the opposite is also the case: scrolling down -> page up)

device info
- android 11
- z flip3

https://user-images.githubusercontent.com/15874357/138772840-8b7e226b-e67d-451f-9009-309b3770de06.mp4

**Comment by derornos** [comment]
My "Z filp3" has now been upgraded to Android 12. But still have the same scrolling issue. As you said, it seems right that this issue is caused by split-loading the data. I hope this problem will be fixed soon.

**Comment by zzorba** [comment]
Yeah, so I completely rebuilt this hoping to use a different scrolling library.  It worked better on IOS, but was unfortunately much worse on Android.  There's a rewrite of it underway by the maintainers which I'm hopeful will address some of the issues.

Still on the lookout for a serviceable base library for react-native that has 1) android support and 2) pull to refresh support.
